partners/project-18.md

```diff
@@ -0,0 +1,59 @@
+---
+altLangPage: "https://conception.canada.ca/partenaires/project-18.html"
+date: 2024-01-03
+dateModified: 2024-01-03
+description: "Partner project: Make it easier for Canadians to make informed choices about vaccination. Date: Fall 2017."
+title: "Disease and immunization information"
+---
+<p>Make it easier for businesses to fulfill their Goods and Services Tax (GST), Harmonized Sales Tax (HST) and payroll requirements.</p>
```

Comment:
This paragraph is incorrect. It should be: "Make it easier for Canadians to interpret immunization guidelines."

Reply:
Fixed.